Libplanet/Net/NetMQTransport.cs

```diff
@@ -857,7 +857,11 @@ private string ToNetMQAddress(BoundPeer peer)
 
         private async Task CreatePermission(BoundPeer peer)
         {
+            var cts = new CancellationTokenSource();
             IPAddress[] ips;
+
+            // Cancellation After 2.5 sec
+            cts.CancelAfter(2500);
```

Comment:
Can we have a test case for this?

I don't think it will work as intended because the `Token` from `cts` is not used anywhere.

Reply:
Still we need a test case for this?

Now `GetMappedAddressAsync` and `CreatePermissionAsync` using `Token` from `cts`.